### DISCUSSION TITLE
Welcome to CARLA Discussions!

### DISCUSSION THREAD

**Discussion by germanros1987** [carla-simulator/carla#4105] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MalikSumbal** [comment]
Dear All 

I am new to Carla Simulator. I would be thankful if you could answer the following questions:

1. Does Carla work best on Windows or Linux?
2. Can we run the Carla simulator without GPU?
3. Is it possible to install and run Carla on Google Cloud?
